primitives/phragmen/Cargo.toml

```diff
@@ -15,13 +15,15 @@ targets = ["x86_64-unknown-linux-gnu"]
 codec = { package = "parity-scale-codec", version = "1.0.0", default-features = false, features = ["derive"] }
 serde = { version = "1.0.101", optional = true, features = ["derive"] }
 sp-std = { version = "2.0.0-dev", default-features = false, path = "../std" }
-sp-runtime = { version = "2.0.0-dev", default-features = false, path = "../../primitives/runtime" }
 sp-phragmen-compact = { version = "2.0.0-dev", path = "./compact" }
+sp-arithmetic = { version = "2.0.0-dev", default-features = false, path = "../arithmetic" }
+sp-core = { version = "2.0.0-dev", default-features = false, path = "../core" }
```

Comment:
Replacing `sp-runtime` with `sp-core` does not solve the issue.

Reply:
yeah indeed, goal is to move this out of substrate and not depend on these beefy crates like `sp-runtime` and `sp-core`. Only arithmetic is really needed. but it may not be as trivial as I formulated in the issue: 

1- custom Debug impls will be needed
2- Some trait bounds (like `Member`) could be removed if needed.

Reply:
1. Is really easy. `RuntimeDebug` just prints `<wasm:stripped>` when calling `Debug`.
2. Is solved.

Reply:
Can we move `sp-debug-derive` outside of substrate too and still use it in order to reduce manual `Debug` implementations?